src/components/ActNowButtonCard/index.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from "react";
+import { Card, CardContent, Typography, Grid, Button } from "@material-ui/core";
+import UpdateFormDialog from "@/promisetracker/components/FormDialog/UpdateDialog";
+
+import AllInclusiveIcon from "@material-ui/icons/AllInclusive";
+import ChatBubbleOutlineIcon from "@material-ui/icons/ChatBubbleOutline";
+import ControlPointIcon from "@material-ui/icons/ControlPoint";
+import NotificationsNoneIcon from "@material-ui/icons/NotificationsNone";
+import ShareIcon from "@material-ui/icons/Share";
+
+import ConnectCard from "./ConnectCard";
+import ShareCard from "./ShareCard";
+import FollowCard from "./FollowCard";
+import PetitionCard from "./PetitionCard";
+
+import useStyles from "./useStyles";
+
+const ActNowButtonCard = ({ ...props }) => {
+  const [open, setOpen] = useState(false);
+
+  const selectedCard = {
+    CONNECT: "connect",
+    FOLLOW: "follow",
+    PETITION: "petition",
+    SHARE: "share",
+    ACT: "act",
+  };
+
+  const [card, setSelectedCard] = useState(selectedCard.ACT);
+
+  const handleFormOpen = () => {
+    setOpen(true);
+  };
+
+  const handleFormClose = () => {
+    setOpen(false);
+  };
+
+  const selectCard = (name) => {
+    setSelectedCard(selectedCard[name]);
+  };
+
+  const classes = useStyles();
+
+  return (
+    <>
```

Comment:
Can't this card also be built on top of `BaseCard` if we make close optional?

Reply:
CardContent is the reusable bit, so far we have only one card, the content is what's dynamic.

Reply:
Cool, cool... so can't we reuse `BaseCard` to replace this `CardContent` [here](https://github.com/CodeForAfrica/PromiseTracker/pull/309/files#diff-288c58d22ceb0758f7e3df61577d06a789fbb83e32367dd2e0de66038850d923R52-R56)?

Reply:
Oh! Yes, that is possible @kilemensi